python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -387,6 +448,62 @@ def run_optimize_model_gpu(repo_id,
     torch.xpu.empty_cache()
     return result
 
+def run_original_model_gpu(repo_id,
+                           local_model_hub,
+                           in_out_pairs,
+                           warm_up,
+                           num_trials):
+    from transformers import AutoModel, AutoModelForCausalLM, AutoTokenizer, GPTJForCausalLM
+    import intel_extension_for_pytorch as ipex
```

Comment:
do we need to import ipex when collect baseline performance data of without using `optimize_model`?

Reply:
Looks like it is unused in code below. I'll remove it.

Reply:
Looks like removing it will cause `to('xpu')` stop working. I've added it now.